Lib/compression/zstd/__init__.py

```diff
@@ -40,6 +40,11 @@
 COMPRESSION_LEVEL_DEFAULT = _zstd.ZSTD_CLEVEL_DEFAULT
 """The default compression level for Zstandard, currently '3'."""
 
+# Set ZSTD_EndDirective constants on ZstdCompressor
+ZstdCompressor.CONTINUE = _zstd.ZSTD_e_continue
+ZstdCompressor.FLUSH_BLOCK = _zstd.ZSTD_e_flush
+ZstdCompressor.FLUSH_FRAME = _zstd.ZSTD_e_end
+
```

Comment:
I can understand why we do this, and how beneficial it is, but this presumes that we should always setup `ZstdCompressor` somehow. For instance, if we only want to check the C class without importing `compression.zstd` but directly `_zstd`, then we wouldn't have those attributes.

Reply:
I've pushed the revert to define the constants in C. I'll open a new PR to make the types immutable, as it doesn't fit neatly into this one.